### ISSUE TITLE
Make the bundle UUID more available for searching (e.g. include in bundle manifest)

### ISSUE THREAD

**Issue by mikebaumann** [HumanCellAtlas/data-store#801]
Searching based on bundle UUID (and version) should be fast and easy. In addition to simply verifying the bundle exists in the DSS, now that output format `raw` is available, it is a convenient way to view all the indexed metadata in the bundle. 

Currently, however, the bundle UUID is not included in the bundle manifest, nor is it explicitly added to the index document. **Is there a reason why the bundle UUID is not included in the bundle manifest, along with the bundle version, which is currently included in the bundle manifest?** It seems having the content of the bundle manifest being self-identifying, with both the bundle UUID and version, would be useful in multiple use cases, not just for search.

It is currently possible to query for a given bundle UUID and version using the Elasticsearch Ids query, but that is crufty, awkward and depends on internal implementation details.

```
{
    "query": {
        "ids" : {
            "values" : ["9f61dcee-13d0-4b9f-8dd4-b80e53e4b895.2017-12-21T174128.247427Z"]
        }
    }
}
```

If there is a good reason for not including the bundle UUID in the bundle manifest, then the bundle  UUID should be explicitly added to the index document content. Yet, I think the former is preferable.

**Comment by Bento007** [comment]
@mikebaumann what is left to be done?

**Comment by hannes-ucsc** [comment]
We should use the `(fixes #123)` or `(resolves #123)` tag in commit (!) titles, so GH closes the ticket as soon as the commit lands on master.

**Comment by hannes-ucsc** [comment]
I think 1c21d736d58d23fa4a2c6fe424cd4ef701bd4956 addresses this.
